configure.ac

```diff
@@ -3,4 +3,24 @@ AM_INIT_AUTOMAKE(ke-recv-extra, 0.2)
 AM_CONFIG_HEADER(config.h)
 AC_PROG_CC
 
+PKG_CHECK_MODULES(GCONF, gconf-2.0)
+AC_SUBST(GCONF_CFLAGS)
+AC_SUBST(GCONF_LIBS)
```

Comment:
It's done done automatically, these lines may be removed

Reply:
It took me a bit of searching, but I found this:

``` 
On older versions of pkg-config, the macro will not call AC_SUBST on these variables; modern versions (at least version 0.24) will take care of that already. Running it twice, though, will not cause problems, so if you have doubts, you should add a snippet like the following. 
```

From: https://autotools.io/pkgconfig/pkg_check_modules.html

I've think we have this almost everywhere. I'm OK with removing it here, if you think it makes sense.

Reply:
From Debian Stretch:
```
pkg-config --version
0.29
```

From https://cgit.freedesktop.org/pkg-config/refs/:
`pkg-config-0.24 ... 10 years`

It's there for at least 10 years. Maybe it's time to start using "new" functionality? :)

Reply:
Done in a separate commit (I didn't want to touch both the commits)